indexer/services/socks/src/lib/workers/forwardMessageWorker.ts

```diff
@@ -0,0 +1,9 @@
+import { parentPort } from 'worker_threads';
+
+import { MessageForwarder } from '../message-forwarder';
+
+parentPort?.on('message', async (data) => {
+  const { subscriptions, index, messageToForward } = data;
+  await MessageForwarder.getInstance(subscriptions, index).forwardMessage(messageToForward);
```

Comment:
As I understand, workers should avoid doing IO and instead only do the CPU-bound part of the job (serializing/deserializing, logging/metrics, etc)